src/main/groovy/com/rundeck/plugins/ansible/ansible/AnsibleDescribable.java

```diff
@@ -527,4 +529,13 @@ public static String[] getValues() {
             .title("Encrypt Extra Vars.")
             .description("Encrypt the value of the extra vars keys.")
             .build();
+
+    Property YAML_DATA_SIZE_PROP = PropertyBuilder.builder()
+            .string(ANSIBLE_YAML_DATA_SIZE)
```

Comment:
you can use an `integer` property instead `string`, so you won't need to parse the value later

Reply:
done... ✅